__tests__/app.test.js

```diff
@@ -0,0 +1,54 @@
+const app = require('../app');
+const request = require('supertest');
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection');
+const data = require('../db/data/test-data')
+
+
+afterAll(() => {
+    return db.end();
+});
+
+beforeEach(() => {
+    return seed(data);
+});
+
+describe("GET /api/topics", () => {
+   describe("STATUS: 200", () => {
+    test("endpoint responds with a 200 status", () => {
+        return request(app).get("/api/topics").expect(200);
+    });
```

Comment:
you can combine these first two tests into one. As you test the status in the .expect(200) - so no need to have a separate test for it